exercises/practice/diamond/.meta/config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "authors": [
+    "vaeng"
+  ],
+  "files": {
+    "solution": [
+      "diamond.cpp",
+      "diamond.h"
+    ],
+    "test": [
+      "diamond_test.cpp"
+    ],
+    "example": [
+      "/.meta/example.cpp",
+      "/.meta/example.h"
```

Comment:
are you sure about the leading slashes?

Reply:
Now that you say it, I would remove them. Although configlet is not complaining.